pkg/compactor/compactor.go

```diff
@@ -121,7 +121,7 @@ type Config struct {
 func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	cfg.ShardingRing.RegisterFlags(f, logger)
 
-	cfg.BlockRanges = DurationList{3 * time.Hour, 6 * time.Hour, 12 * time.Hour}
+	cfg.BlockRanges = DurationList{1 * time.Hour, 4 * time.Hour, 8 * time.Hour}
```

Comment:
In the description it's said the block ranges are to be: 1, 2, and 8 hours. I don't know exactly what is most optimal configuration, but assume that the earlier blocks get compacted, the better, therefore the 1, 2, 8 setup intuitively looks more appealing

Reply:
I think it's fair to have the 2h mark in case the 4h is very slow. I will change it.